pyintegration/deephaven2/combo_agg.py

```diff
@@ -0,0 +1,277 @@
+#
+#  Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+
+from typing import List, Any
+
+import jpy
+
+ComboAggregateFactory = jpy.get_type("io.deephaven.db.v2.by.ComboAggregateFactory")
```

Comment:
The usage of this looks like:

```python
from deephaven import ComboAggregation
x = table.combo_by(ComboAggregation().sum("Bar", "Baz").median("Baz"), ["Foo"])
```
if I'm looking correctly?

I think `io.deephaven.api.agg.Aggregation` may be the better interface to use, and we can wrap the Aggregation instead of the `ComboAggregation`. Then we could have: 

```python
def combo_by(self, by: List[str], aggs: List[Aggregation])
```

```python
from deephaven.Aggregation import sum, median

x = table.combo_by(["Foo"], [sum("Bar", "Baz"), median("Baz")])
```

Reply:
yes, that's the correct usage, and it is the same interface as the one in the client (pydeephaven).  I do like the simpler interface but I also like the explicitness and slightly better readability of ComboAggregation.  Need to hear @chipkent's thoughts.

Reply:
yes, that's the correct usage, and it is the same interface as the one in the client (pydeephaven). I do like the simpler interface but I also like the explicitness and slightly better readability of ComboAggregation. Need to hear @chipkent's thoughts.

Reply:
1) I think the jpy type should be private if it is there.
2) I think the `Aggregation` syntax is cleaner than the `ComboAggregation` syntax.
3) I assume the module would be `deephaven.aggregation` and not `deephaven.Aggregation`.